findme-location-riddles/src/entities/LocationRiddle.py

```diff
@@ -1,23 +1,11 @@
 from datetime import datetime
-
-from pydantic import BaseModel
+from pydantic import BaseModel, field_validator, ValidationError
 from typing import List, Optional
 from decimal import Decimal
 
-
-class Rating(BaseModel):
-    user_id: str
-    rating: int
-
-
-class Comment(BaseModel):
-    user_id: str
-    comment: str
-
-
-class Guess(BaseModel):
-    user_id: str
-    guess: List[Decimal]
+from src.entities.Rating import Rating
```

Comment:
at some point we could move all entities to the shared code as we might want to use these models in other micro services later, not urgent of course